### PR TITLE
Misc framer fixes

### PR DESCRIPTION
**Pull request by eps1lon**
Misc changes that are not suitable for GitHub Suggestions:

* declare dependencies explicitly
  This is in part to avoid subtle defects currently and prepare for yarn v2 where implicit dependencies will throw. This might seem purely academic but allows some pretty significant optimizations and avoids confusion if errors are thrown from these dependencies (e.g. which version is this even).
* `devDependencies` -> `dependencies` for private workspaces (cant find the explainer in the yarn docs anymore but TL;DR: devDep for local deps, deps for published deps)
* some cli best practices in the form of using process.cwd if resolving passed arguments (if you know where the script is located then it's kind of Ok but you don't want to go through mental gymnastics when deciding where inputs/outputs should be located.)
* Not sure where the ListItem change comes from. I just ran `yarn build` inside the framer workspace
* use the same babel environment as `yarn docs:api`. Not sure why it doesn't transpile the defaultPropsResolver but we want to get away from this anyway (since it has no benefit) so just converting to commonJS is fine.

### CODE REVIEW

File: framer/package.json [hunk 1]
```diff
@@ -7,13 +7,16 @@
...
   "license": "MIT",
   "private": true,
-  "devDependencies": {
+  "dependencies": {
     "case": "1.6.2",
-    "docs": "^4.0.0"
+    "deepmerge": "^4.0.0",
+    "docs": "^4.0.0",
+    "mustache": "^3.1.0",
+    "react-docgen": "^5.0.0-beta.1"
```
Comment (eps1lon) [inline framer/package.json]: These were used in `buildFramer.js`

File: framer/Material-UI.framerfx/code/ListItem.tsx [hunk 2]
```diff
@@ -87,6 +87,30 @@ export const ListItem: React.SFC<Props> = (props: Props) => {
...
+      break;
+    case 'checkbox':
+      primary = (
+        <MuiListItemIcon>
+          <MuiCheckbox />
+        </MuiListItemIcon>
+      );
+      break;
+  }
+
```
Comment (mbrookes) [inline framer/Material-UI.framerfx/code/ListItem.tsx]: Duplicated?

Comment (mbrookes) [reply framer/Material-UI.framerfx/code/ListItem.tsx]: Ah, sorry. My mistake. Could you rebase and re-run the script?

Comment (eps1lon) [reply framer/Material-UI.framerfx/code/ListItem.tsx]: Yep this was duplicated in the template.

**Comment by mbrookes** [comment]
Thanks!

**Pull request merged**